steps/azure_web_app_deploy.yml

```diff
@@ -96,10 +99,10 @@ steps:
       else
         echo "##[command]Sending webhook... App Service will pull latest version of tag: $(ENVIRONMENT)"
         
-        STATUSCODE=$(curl --silent --output /dev/stderr --write-out "%{http_code}" "$CD_URL" -X POST -H "Content-Length:0")
+        STATUS_CODE=$(curl --silent --output /dev/stderr --write-out "%{http_code}" "$CD_URL" -X POST -H "Content-Length:0")
 
-        if test $STATUSCODE -ne 200; then
-          echo "##vso[task.logissue type=error]Error sending webhook, web app has not pulled the latest image..."
+        if [[ $STATUS_CODE -lt 200 ]] || [[ $STATUS_CODE -gt 299 ]]; then
```

Comment:
do we want to check if the status is any 200 status potentially?

Reply:
That's what this is doing, checking if the number is in the range 200-299.